live-examples/css-examples/logical-properties/max-block-size.html

```diff
@@ -0,0 +1,41 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="max-block-size">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">max-block-size: 150px;
+writing-mode: horizontal-tb;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">max-block-size: 150px;
+writing-mode: vertical-rl;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">max-block-size: 20px;
+writing-mode: horizontal-tb;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
```

Comment:
I don't think we need this option, here and in the other related examples. It's essentially just option 1.

Reply:
The intention was to keep it consistent with max-height/width and show that it would cause content to be hidden. I think that is useful.

Reply:
OK, that makes sense.